source/draw/DrawState.ooc

```diff
@@ -39,6 +41,21 @@ DrawState: cover {
 		this viewport = viewport
 		this
 	}
+	setDestination: func (destination: IntBox2D) -> This {
```

Comment:
OK, I think it could be good to include this information somewhere in the source code.